petab/visualize/plotting_config.py

```diff
@@ -1,6 +1,7 @@
 """Plotting config"""
 import numpy as np
 import pandas as pd
+import seaborn as sns
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'seaborn'](https://app.codacy.com/manual/dweindl/PEtab/pullRequest?prid=4955484)